spec/features/book_show_spec.rb

```diff
@@ -16,8 +16,8 @@
       expect(page).to have_content("Title: Where The Wild Things Are")
       expect(page).to have_content("Page Count: 40")
       expect(page).to have_content("Year Published: 1987")
-      expect(page).to have_content("Author 1 name")
-      expect(page).to have_content("Author 2 name")
+      expect(page).to have_link("Author 1 name", href: author_path(author_1))
+      expect(page).to have_link("Author 2 name", href: author_path(author_2))
```

Comment:
I did not know about being able to call the href in the expects. That's going to be really handy.